superset-frontend/src/views/CRUD/data/dataset/AddDataset/index.tsx

```diff
@@ -73,6 +75,38 @@ export default function AddDataset() {
     Reducer<Partial<DatasetObject> | null, DSReducerActionType>
   >(datasetReducer, null);
   const [hasColumns, setHasColumns] = useState(false);
+  const [linkedDatasets, setLinkedDatasets] = useState<DatasetObject[]>([]);
```

Comment:
this is a nit, but has been tripping me up when reading the code. Just for naming purposes, this is a list of all datasets, right? I'm not clear on which part is linked.

Reply:
These are the datasets that are linked to a table, I wasn't sure of a great name for this. What would be a better name?

Reply:
how about just "physicalDatasets"? the name doesn't explain what they're linked to, so maybe physicalDatasets will  explain that they are just regular datasets, but only the physical, not virtual, ones.. honestly you could probably just say "datasets" and that would be fine, too. I don't think it's necessary to completely describe them here.. just that it's a list of datasets should be fine.

Reply:
Changed all instances of `linkedDatasets` to `datasets` in [`this commit`](https://github.com/apache/superset/pull/22136/commits/ad13ff4dbb6c72d647397d3b2d106c6fc3dbbe2c).

Reply:
Looks good!